src/js/reducers/layoutReducer.ts

```diff
@@ -0,0 +1,64 @@
+import {Map} from 'immutable';
```

Comment:
for consistency with the rest of the reducer filenames, i would name this `layoutsReducer.ts` (layouts plural). right-click rename in vscode should prompt you to automatically update all imports. do this in a fresh commit so you don't mix up the auto-rename with other changes in case something goes wrong